src/nvim/event/process.h

```diff
@@ -23,13 +23,15 @@ struct process {
   uint64_t stopped_time;
   const char *cwd;
   char **argv;
-  Stream *in, *out, *err;
+  Stream in, out, err;
```

Comment:
That explains gcc:s strange warning message (it complains of missing braces for all of Process, not the Stream members).